src/Solvers/conjugate_gradient_poisson_solver.jl

```diff
@@ -31,36 +31,54 @@ end
 
 @kernel function laplacian!(∇²ϕ, grid, ϕ)
     i, j, k = @index(Global, NTuple)
-    @inbounds ∇²ϕ[i, j, k] = ∇²ᶜᶜᶜ(i, j, k, grid, ϕ)
+    active = !inactive_cell(i, j, k, grid)
+    @inbounds ∇²ϕ[i, j, k] = ∇²ᶜᶜᶜ(i, j, k, grid, ϕ) * active
+end
+
+struct RegularizedLaplacian{D}
+    δ :: D
 end
 
-function compute_laplacian!(∇²ϕ, ϕ)
+function (L::RegularizedLaplacian)(Lϕ, ϕ)
     grid = ϕ.grid
     arch = architecture(grid)
     fill_halo_regions!(ϕ)
-    launch!(arch, grid, :xyz, laplacian!, ∇²ϕ, grid, ϕ)
+    launch!(arch, grid, :xyz, laplacian!, Lϕ, grid, ϕ)
+
+    if !isnothing(L.δ)
+        # Add regularization
+        ϕ̄ = mean(ϕ)
+        ΔLϕ = L.δ * ϕ̄
+        grid = ϕ.grid
+        arch = architecture(grid)
+        launch!(arch, grid, :xyz, subtract_and_mask!, Lϕ, grid, ΔLϕ)
+    end
+
     return nothing
 end
 
 struct DefaultPreconditioner end
 
 function ConjugateGradientPoissonSolver(grid;
+                                        regularization = nothing,
```

Comment:
I think it would be clearer to call this the `regularization_constant`. What do you think?

Reply:
How about `regularizer`?

Reply:
Works for me!